build/pipelines/templates/extensions-variables.yml

```diff
@@ -1,10 +1,10 @@
- variables:
-    ${{ if not(contains(variables['Build.SourceBranch'], '/release/' )) }}:
-      buildNumberTemp: $(Build.BuildNumber)
-    ${{ if contains(variables['Build.SourceBranch'], '/release/' ) }}:
-      isReleaseBuildTemp: true
-    buildNumber: $[variables.buildNumberTemp]
-    isReleaseBuild: $[variables.isReleaseBuildTemp]
-    solution: '**/*.sln'
-    buildPlatform: 'Any CPU'
-    buildConfiguration: 'Release'
\ No newline at end of file
+variables:
+  ${{ if not(contains(variables['Build.SourceBranch'], '/release/' )) }}:
+    buildNumberTemp: $(Build.BuildNumber)
+  ${{ or(contains(variables['Build.SourceBranch'], '/release/'), startsWith(variables['Build.SourceBranch'], 'refs/tags')) }}:
```

Comment:
It's triggered automatically.  I don't think we need to change anything else - temp builds are just the ones that run on PRs/manual triggers and they're differentiated by the builder number. You know it's not temp because the triggering branch is release or (as of this PR) tagged